Vector.c

```diff
@@ -293,13 +285,19 @@ void Vector_compact(Vector* this) {
    /* one deletion: use memmove, which should be faster */
    if (this->dirty_count == 1) {
       memmove(&this->array[idx], &this->array[idx + 1], (this->items - idx - 1) * sizeof(this->array[0]));
+      this->array[this->items - 1] = NULL;
    } else {
       /* multiple deletions */
       for (int i = idx + 1; i < size; i++) {
          if (this->array[i]) {
             this->array[idx++] = this->array[i];
          }
       }
+      /* idx is now at the end of the vector and on the first index which should be set to NULL */
```

Comment:
-
```suggestion
      // idx is now at the end of the vector and on the first index which should be set to NULL
```

Also note the brace style in the `for` loop below.

Reply:
Should i change the style of the other comments in this function too?